bundles/org.openhab.binding.semsportal/README.md

```diff
@@ -0,0 +1,45 @@
+# SEMSPortal Binding
+
+This binding can help you include statistics of your SEMS / GoodWe solar panel insallation into openHAB. It is a readonly connection that maps collected parameters to openhab channels. It provides current, day, month and total yields, as well as some income statistics if you have configured these in the SEMS portal. It requires a power station that is connected through the internet to the SEMS portal.
+
+## Supported Things
+
+There is only one thing type supported in this binding, which is the powerStation. You need to provide the identifying UUID of the solar system in your account yourself. If you have multiple powerstations that report to the same account, you can configure the same account for each powerSation and only differ the unique id of the station in each Thing.
+
+## Discovery
+
+At this moment there is no discovery possible. Configuration of the powerStation unique id is done by hand in the Thing configuration page.
+
+## Thing Configuration
+
+The configuration of the thing is pretty straight forward. You need to have your power station set up in the SEMS portal, and you need to have an account that is allowed to view the power station data. You should log in at least once in the portal with this account to activate it. 
+
+The thing needs the username and password to connect and retreive the data. It also needs the unique id of the power station. It can be found after you log in to the portal, on the URL. use the value you find in the URL at the x-es (including the -s) in this example: https://www.semsportal.com/powerstation/powerstatussnmin/xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+
+
+## Channels
+
+
+| channel        | type      | description                                                                                                |
+| -------------- | --------- | ---------------------------------------------------------------------------------------------------------- |
+| online_state   | Switch    | If the powerStation is currently on and reporting to the portal                                            |
```

Comment:
Channels should use mixedCase naming convention. Example onlineState

Reply:
Ok I'm mainly experienced with ones that don't like zwave (switch_dimmer, alarm_motion etc). modified to camelCase.

Reply:
Yes it is annoying when you follow merged examples and then get told differently when getting a review. Last night I looked through the dev guidelines and found broken links and feel this area needs work in the documentation as 3 of the latest binding submissions have all done this and it was only the 2.1 version guidelines that were actually clear on this at least in my opinion.